packages/tailwindcss/src/css-functions.ts

```diff
@@ -142,3 +218,22 @@ function eventuallyUnquote(value: string) {
 
   return unquoted
 }
+
+function injectFallbackForInitialFallback(ast: ValueParser.ValueAstNode[], fallback: string): void {
+  ValueParser.walk(ast, (node) => {
+    if (node.kind !== 'function') return
+    if (node.value !== 'var' && node.value !== 'theme' && node.value !== '--theme') return
+
+    if (node.nodes.length === 1) {
+      node.nodes.push({
+        kind: 'word',
+        value: `, ${fallback}`,
+      })
```

Comment:
This seems _fine_ but don't we technically treat the `,` as a separate node when parsing? Should we do that here too?

Reply:
yeah we do, the fallback could also consist of other, nested, functions etc. It's technically more correct to parse the fallback as well but it felt super wasteful to parse it just so it's stringified in the next line. If we ever change stuff so that the stringification won't work anymore if the word node contains stuff it shouldn't, we also have unit tests that would let us know this breaks so I decided to just put it in as a `word` node.

Reply:
👍